convert2rhel/pkghandler.py

```diff
@@ -500,3 +501,14 @@ def install_additional_rhel_kernel_pkgs(additional_pkgs):
             loggerinst.info("Installing RHEL %s" % name)
             call_yum_cmd("install %s" % name)
     return
+
+
+def check_rhn_classic():
+    loggerinst = logging.getLogger(__name__)
+    if os.path.isfile("/etc/sysconfig/rhn/systemid"):
+        loggerinst.critical("RHN Classic detected! \n"
+                            "Please follow the bellow link for more information how to disable it:\n"
+                            "https://access.redhat.com/documentation/en-us/red_hat_subscription_management/1/html/migrating_from_rhn_classic/index"  # pylint: disable=C0321
+                            "\n\n Convertion will not proceed.")
+
+    return
```

Comment:
I think `return` here is unnecessary

Reply:
We should be consistent with everywhere else, code still reaches the return so we should keep it